From 2b8d18868878a327d88d65512463c235eb142244 Mon Sep 17 00:00:00 2001
From: Rene Engelhard <rene@debian.org>
Date: Tue, 11 Jul 2023 16:56:02 +0200
Subject: [PATCH] add linux_riscv64 and linux_loongarch64 to dp_platform.cxx

forgotten in bc9487f745befde6534fd46058e119256952323d and d3625d968901eb93a9680db8d1165f70de3fd64e

Change-Id: I7f33c1db54b8f66e797a29cd1ccf96d19a88cc60
---
 desktop/source/deployment/misc/dp_platform.cxx | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/desktop/source/deployment/misc/dp_platform.cxx b/desktop/source/deployment/misc/dp_platform.cxx
index ad1b5ea13fed..b2af59f9b926 100644
--- a/desktop/source/deployment/misc/dp_platform.cxx
+++ b/desktop/source/deployment/misc/dp_platform.cxx
@@ -126,6 +126,10 @@ namespace
             ret = checkOSandCPU(u"Linux", u"ALPHA");
         else if (token == u"linux_aarch64")
             ret = checkOSandCPU(u"Linux", u"AARCH64");
+        else if (token == u"linux_riscv64")
+            ret = checkOSandCPU(u"Linux", u"RISCV64");
+        else if (token == u"linux_loongarch64")
+            ret = checkOSandCPU(u"Linux", u"LOONGARCH64");
         else if (token == u"freebsd_x86")
             ret = checkOSandCPU(u"FreeBSD", u"x86");
         else if (token == u"freebsd_x86_64")
-- 
2.39.2

